.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+# Elixir CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-elixir/ for more details
+version: 2
+jobs:
+  build:
+    working_directory: ~/repo
+    docker:
+      - image: defactosoftware/elixir:1.5
+        environment:
+          MIX_ENV: test
+    steps:
+      - run: mix local.hex --force
+      - run: mix local.rebar --force
+      - checkout
+      - restore_cache:
```

Comment:
you need a step to save these as well, otherwise this has no use...

Reply:
a right!